compute/cker/include/cker/train/operation/DepthwiseConv.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __NNFW_CKER_TRAIN_OPERATION_DEPTHWISECONV_H__
+#define __NNFW_CKER_TRAIN_OPERATION_DEPTHWISECONV_H__
+
+#include "cker/eigen/depthwise_conv_op.h"
+#include "cker/Shape.h"
+#include "cker/Types.h"
+
+namespace nnfw
+{
+namespace cker
+{
+namespace train
+{
+
+class DepthwiseConv
+{
+public:
+  DepthwiseConv() = default;
+
+  template <typename T> int64_t kPacketSize() const
+  {
+    typedef typename Eigen::internal::packet_traits<T>::type Packet;
```

Comment:
(Optional) You can replace `typedef` with _alias declaration_
```suggestion
    using packet_t = typename Eigen::internal::packet_traits<T>::type;
```

Reply:
I checked that this code was from tensorflow. You can ignore the above comment.

Reply:
This code also came from TensorFlow. How about keeping the existing code for maintaining?

Reply:
Isn't this code in this file that can be modified?
Anyway, this is optional.

Reply:
I prefer to keep the existing code for maintaining issue. The cker code almost(?) came from TensorFlow. Even a simple expression may be difficult to search if it does not match. For this reason, I copied the TensorFlow code with a minimum of modification.